changelogs/fragments/60508-route53-improve-octal-characters-handling.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+- route53 - improve handling of octal encoded characters
```

Since you mentioned it explicitly:
```suggestion
- route53 - improve handling of octal encoded characters, for example for ``#``.
```